install_magenta.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
```

Comment:
I'd suggest using 'set -e' at the top because we probably don't want to continue if any step fails.